checkov/common/graph/checks_infra/enums.py

```diff
@@ -33,6 +33,10 @@ class Operators:
     NOT_EQUALS = 'not_equals'
     REGEX_MATCH = 'regex_match'
     NOT_REGEX_MATCH = 'not_regex_match'
+    GREATER_THAN = 'greater_than'
```

Comment:
can we add a test that validates what happens with those operators on string, int and map values?

Reply:
I have tests for all the combinations of string / int / float.

Anything that can't be converted to a float is compared as a string. The behavior is undefined for if it's a map or list. I will add a test to sanity check that it doesn't break.